src/labelformat/cli/cli.py

```diff
@@ -65,27 +65,13 @@ def main() -> None:
         # Add input/output format arguments.
         convert_parser.add_argument(
             "--input-format",
-            choices=sorted(
-                set(
-                    name
-                    for tsk, name_to_cls in _REGISTRY.input.items()
-                    for name in name_to_cls
-                    if tsk == task
-                )
-            ),
+            choices=sorted(_REGISTRY.input[task].keys()),
```

Comment:
I'm not sure anymore why we ever wrote this complex code. This is just a simplification.